Test2/Tests/CalculatorPageTests.cs

```diff
@@ -0,0 +1,268 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+
+namespace Test2.Tests
+{
+    internal class CalculatorPageTests
+    {
+        public IWebDriver driver;
+
+        [SetUp]
+        public void SetUp()
+        {
+            var options = new ChromeOptions { AcceptInsecureCertificates = true };
+            driver = new ChromeDriver(options);
+            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(60);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+            driver.Url = "https://localhost:5001/Calculator";
+        }
+        [TearDown]
+        public void TearDown()
+        {
+            driver.Quit();
+        }
+
+
+        [Test]
+        public void Names()
+        {
+            IWebElement depAm = driver.FindElement(By.XPath("(*//td[contains ( text(), 'Deposit' )])"));
+            IWebElement rateInt = driver.FindElement(By.Id("percent"));
+            IWebElement term = driver.FindElement(By.Id("term"));
+            IWebElement startDay = driver.FindElement(By.Id("day"));
+            IWebElement startMonth = driver.FindElement(By.Id("month"));
+            IWebElement startYear = driver.FindElement(By.Id("year"));
+            IWebElement calcBut = driver.FindElement(By.Id("calculateBtn"));
+            IWebElement termBut = driver.FindElement(By.XPath("//input[@type='radio']"));
+            SelectElement startDaySelect = new SelectElement(startDay);
+
+            rateInt.SendKeys("100");
+            term.SendKeys("365");
+            //startDay.SendKeys("01");
+            startDaySelect.SelectByText("1");
+            startMonth.SendKeys("January");
+            startYear.SendKeys("2022");
+            ////*[contains ( text(), '365 days' )]/input
+            termBut.Click();
+            calcBut.Click();
+            Thread.Sleep(600);
+            IWebElement endDay = driver.FindElement(By.Id("endDate"));
+            //XPath("//*[contains ( text(), 'End Date' )]/..//input"));
+            /*new WebDriverWait(driver, TimeSpan.FromSeconds(10))
+            .Until(ExpectedConditions.TextToBePresentInElement(er, expected));*/
+            string expected = "Deposit amount: *";
+            Thread.Sleep(500);
+            Assert.AreEqual(expected, depAm.Text);
+
+        }
+        [Test]
```

Comment:
Please add a blank line between tests.
```suggestion
        }
        
        [Test]
```